lib/collective-sections.js

```diff
@@ -180,6 +184,39 @@ export const filterSectionsByData = (sections, collective, isAdmin, isHostAdmin)
     }
   }
 
+  // New Connect section
+  // Don't display if:
+  // 1. both Conversations and Updates are disabled
+  // 2. only Updates is enabled but the user viewing is a non-admin user
+  const HAS_NEW_COLLECTIVE_NAVBAR = parseToBoolean(getEnvVar('NEW_COLLECTIVE_NAVBAR'));
```

Comment:
Surprising to have the variable as uppercase/constant compared to the other ones below

Reply:
Resolved - good to merge otherwise?